en/docs/release-notes/version-3.3.0.md

```diff
@@ -39,4 +39,4 @@ If you are new to Ballerina, you can download the [installers](/downloads/#swanl
 - It also comes with a view to see record-type compositions and relationship.
 - Use the "Ballerina: Project Design" command to open design view.
 
-Apart from the above, there are lots of other bug fixes and improvements. For more details, see the [issue list](https://github.com/wso2/ballerina-plugin-vscode/issues?q=is%3Aissue+is%3Aclosed). 
+For the other bug fixes and improvements that were done, see the [issue list](https://github.com/wso2/ballerina-plugin-vscode/issues?q=is%3Aissue+is%3Aclosed). 
```

Comment:
Why are we pointing to the full issue list without any milestone filter? Current URL lists all the fixes done in the entire history.

Reply:
@gigara @keizer619 - Can we have the milestone link for the 3.3.0 release, please?

Reply:
Currently we don't have milestones previous releases and we have just started milestones from [current release only](https://github.com/wso2/ballerina-plugin-vscode/milestones)

Since its a previous one, shall we remove this section and include from next release onwards?

Reply:
Removed.